lib/stm32wba/hci/README

```diff
@@ -85,8 +85,8 @@ Dependencies:
 URL:
    https://github.com/STMicroelectronics/STM32CubeWBA
 
-commit:
-   v1.2.0
+Commit:
+   d2bde9968677fdfd302e2b595239721e3c178e88
 
```

Comment:
Please add the following in patch list section:
   *Changes from official delivery:
    -dos2unix applied
    -trailing white spaces removed